docs/_static/rotkehlchen_docs.css

```diff
@@ -1,6 +1,11 @@
 .wy-side-nav-search > a {
   width: 175px;
 }
+
 .red {
   color: red;
 }
+
+.rst-content section > img {
+  margin-bottom: 24px !important;
```

Comment:
what does this do?

Reply:
this add margin after every images.

Before
<img width="763" alt="image" src="https://user-images.githubusercontent.com/26648140/214474667-f2a90694-310e-4749-abc3-e17da50f83f6.png">

After
<img width="776" alt="image" src="https://user-images.githubusercontent.com/26648140/214474725-4dc26e20-82b2-40ad-b3e1-d9cf6279d5fa.png">